simulator/src/main/java/module-info.java

```diff
@@ -3,6 +3,7 @@
 /** Runtime module of the simulator. */
 module com.hedera.block.simulator {
     exports com.hedera.block.simulator.config.data;
+    exports com.hedera.block.simulator;
```

Comment:
IntelliJ warns me when I use the classes after this export here.

Here some reference: https://stackoverflow.com/questions/54755914/java-10-intellij-warning-class-is-not-exported-accessing-in-same-module

I am not sure how relevant this is, maybe we should check it out?